app/models/institution_builder.rb

```diff
@@ -641,26 +641,28 @@ def self.add_extension_campus_type(version_id)
 
   def self.add_sec103(version_id)
     str = <<-SQL
+      -- set default message for IHL institutions
+      UPDATE institutions SET section_103_message = 'No information available at this time'
+      FROM weams
+      WHERE weams.facility_code = institutions.facility_code
+        AND SUBSTRING(weams.facility_code, 1, 2) IN('11', '12', '13', '21', '22', '23', '31', '32', '33')
+        AND institutions.version_id = #{version_id};
+
       -- set message based on sec103s
```

Comment:
Will the leave the data in the right state for existing records that already have the ''No information available at this time' message from the original default?

Reply:
Yes, it's fine. We're not concerned about the value of this field in institutions from previous version generation runs.

Reply:
Fair enough.